api/src/main/java/gov/cms/ab2d/api/controller/ErrorHandler.java

```diff
@@ -37,6 +42,16 @@
         return generateFHIRError(e, HttpStatus.BAD_REQUEST);
     }
 
+    @ExceptionHandler({MissingTokenException.class, InvalidAuthHeaderException.class})
+    public ResponseEntity<JsonNode> handleUnauthorizedAccessExceptions(final RuntimeException e) throws IOException {
+        return generateFHIRError(e, HttpStatus.UNAUTHORIZED);
+    }
+
+    @ExceptionHandler({BadJWTTokenException.class, UserNotFoundException.class, UserNotEnabledException.class})
+    public ResponseEntity<JsonNode> handleForbiddenAccessExceptions(final RuntimeException e) throws IOException {
+        return generateFHIRError(e, HttpStatus.FORBIDDEN);
```

Comment:
Since AuthN/Z in our case is out of scope of the Bulk API spec, we don't need to return anything other than the HTTP code itself.